docs/userguide/modules/creating-service-bindings/pages/custom-path-injection.adoc

```diff
@@ -132,3 +132,84 @@ metadata:
+
+== Workload resource mapping
+
+[NOTE]
+====
+. Workload resource mapping is available for the secondary workloads of the ServiceBinding custom resource (CR) for both the API groups: `binding.operators.coreos.com` and `servicebinding.io`.
+. You must namespace `ClusterWorkloadResourceMapping` resources under the `servicebinding.io` API group.  However, the `ClusterWorkloadResourceMapping` resources interact with `ServiceBinding` resources under both the `binding.operators.coreos.com` and `servicebinding.io` API groups.
+====
+
+The specification also provides a way to define exactly where binding data needs to be projected.  Use this method when you are not able to configure custom path locations correctly by previous methods.  You can define `ClusterWorkloadResourceMapping` resources only in the `servicebinding.io` API group, which specify where binding data is to be projected for a given workload kind.
+
+.Example: Schema of the `ClusterWorkloadResourceMapping` resource
+[source,yaml]
+----
+apiVersion: servicebinding.io/v1alpha3
+kind: ClusterWorkloadResourceMapping
+metadata:
+  name: # string <1>
+  ...
+spec:
+  versions: # required, at least one entry must be defined
+  - version: # string <2>
+    annotations: # string (Fixed JSONPath), optional <3>
+    containers: # optional
+    - path: # string (JSONPath) <4>
+      name: # string (Fixed JSONPath), optional <5>
+      env: # string (Fixed JSONPath), optional <6>
+      volumeMounts: # string (Fixed JSONPath), optional <7>
+    volumes: # string (Fixed JSONPath), optional <8>
+----
+<1> Name of the `ClusterWorkloadResourceMapping` resource, which must be qualified as the `plural.group` of the mapped workload resource.  For example, `cronjobs.batch` for CronJobs.
+<2> Version of the resource that is being mapped.  Any version that is not specified can be matched with the "*" wildcard.
+<3> Optional: Identifier of the `.annotations` field in a pod, specified with a fixed JSONPath.  The default value is `.spec.template.spec.annotations`.
+<4> Identifier of the `.containers` and `.initContainers` fields in a pod, specified with a JSONPath.  If no entries under the `containers` field are defined, the {servicebinding-title} defaults to two paths: `.spec.template.spec.containers[*]` and `.spec.template.spec.initContainers[*]`, with all other fields set as their default.  However, if you specify an entry, then you must define the `.path` field.
+<5> Optional: Identifier of the `.name` field in a container, specified with a fixed JSONPath.  The default value is `.name`.
+<6> Optional: Identifier of the `.env` field in a container, specified with a fixed JSONPath.  The default value is `.env`.
+<7> Optional: Identifier of the `.volumeMounts` field in a container, specified with a fixed JSONPath.  The default value is `.volumeMounts`.
+<8> Optional: Identifier of the `.volumes` field in a pod, specified with a fixed JSONPath.  The default value is `.spec.template.spec.volumes`.
+
+[IMPORTANT]
```

Comment:
Question: Is this admonition only for L172?

Suggestion: If yes, then please put `+` in L173, so that the admonition is indented under the Example section.

Reply:
The `[IMPORTANT]` section is supposed to be for the entire list of callouts preceding it, not just for the last one.